app/models/work.rb

```diff
@@ -26,7 +26,7 @@ class Work < ApplicationRecord
     world: 'world'
   }
 
-  state_machine initial: :first_draft do
+  state_machine initial: :initial do
```

Comment:
The state machine changes seem like a key part of this PR. Can you say more about how changing the initial state factors in here?

Reply:
We need to trigger an action when a draft is made, but there's no unique transition we can hook into without creating a new initial state.

Reply:
@jcoyne OK, thanks!